content/en/events/kcsna2021/faq.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Frequently Asked Questions (FAQ)"
+type: docs
+weight: 3
+---
+
+- [When is the summit taking place?](#when-is-the-summit-taking-place)
+- [Where is the summit located?](#where-is-the-summit-located)
+- [Will I be able to attend remotely?](#will-i-be-able-to-attend-remotely)
+- [Why is vaccination required to attend in-person?](#why-is-vaccination-required-to-attend-in-person)
+- [Why do I need to register for KubeCon to attend?](#why-do-i-need-to-register-for-kubecon-to-attend)
+- [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
+- [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
+- [I am not an Org member, can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
+- [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [I have a question! How do I contact the event staff?](#i-have-a-question-how-do-i-contact-the-event-staff)
+
+### When is the summit taking place?
+
+October 11th, the Monday before [KubeCon North America].
+
+[KubeCon]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/
+
+### Where is the summit located?
+
+The in-person location at the
+<a href="https://www.marriott.com/hotels/travel/laxjw-jw-marriott-los-angeles-la-live/" rel="noopener noreferrer" target="_blank">
+JW Marriott LA Live</a>, a 5 minute walk away from the LA Convention Center. See
+the [location] section for more information.
+
+[location]: /events/kcsna2021/location/
+
+
+### Will I be able to attend remotely?
+
+Yes! We are planning a full hybrid event that **should** allow those
+participating remotely to interact and work with contributors on-site. We will
+post more details on how this will work closer to the event.
+
+
+### Why is vaccination required to attend in-person?
+
+As a co-located event the summit is required to adhere to KubeCon’s
+[Health & Safety Guidelines]. There may be additional restrictions that develop
+closer to the event, and we will update the event accordingly to follow venue,
+municipality and CDC guidelines at a minimum.
+
+[Health & Safety Guidelines]: https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/attend/health-and-safety/
```

Comment:
Should this pop to a new tab like on the location page?

Reply:
yeah, I'll take care of that 👍